site/domains.md

```diff
@@ -85,6 +85,8 @@ running in Kubernetes.
 * _Host Path Persistent Volumes:_ If using a `hostPath` persistent volume, then it must be available on all worker nodes in the cluster and have read/write/many permissions for all container/pods in the WebLogic Server deployment.  Be aware
   that many cloud provider's volume providers may not support volumes across availability zones.  You may want to use NFS or a clustered file system to work around this limitation.
 
+* _Security Note:_ The `USER_MEM_ARGS` environment variable defaults to `-Djava.security.egd=file:/dev/./urandom` in all WebLogic server pods and the WebLogic introspection job. It can be explicitly set to another value in your domain resource yaml using the `env` attribute under the `serverPod` configuration.
```

Comment:
in all WebLogic server -> in all WebLogic Server
domain resource yaml -> domain resource YAML file